src/style.css

```diff
@@ -0,0 +1,76 @@
+html {
+  font-family: "Arial", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset your styles with * selector, also you only need to reset body margins in this task